src/utils/url-utils.ts

```diff
@@ -57,3 +57,35 @@ const prepareTilesetUrl = (parsedUrl) => {
     .concat("layers/0");
   return `${parsedUrl.origin}${replacedPathName}${parsedUrl.search}`;
 };
+
+export const viewStateToUrlParams = (viewState, setSearchParams) => {
+  const search = Object.fromEntries(
+    new URLSearchParams(window.location.search)
+  );
+  const { longitude, latitude, pitch, bearing, zoom } = viewState.main;
+  setSearchParams(
```

Comment:
I don't like having a state setter outside of react. I was about returning some ready to use object and set it in the component.
We can have it here but need to make a TSDoc comments anyway.

Reply:
Done